packages/sns/src/sns.wrapper.ts

```diff
@@ -453,6 +454,10 @@ export class SnsWrapper {
     params: SnsNeuronAutoStakeMaturityParams,
   ): Promise<void> => this.governance.autoStakeMaturity(params);
 
+  // Always certified
+  addMaturity = (params: SnsNeuronAddMaturityParams): Promise<bigint> =>
```

Comment:
I would not add it to the wrapper. Or create a new test wrapper.

But we could use the XxxCanister directly for the test methods instead of going the normal route.

Reply:
Or add a test wrapper only.

Reply:
The test canister was added in another pr https://github.com/dfinity/ic-js/pull/412